app/routes/user.py

```diff
@@ -3,7 +3,9 @@
 from app.models.User import User
 from app.server import server
 
-from flask import abort
+
+from flask import g, request, redirect, url_for, abort
+from app.session.csrf import csrf_protected
```

Comment:
need to import user controller

Reply:
user controller is imported in line 2 (github diff hides it)

Reply:
GitHub made me look foolish - I will not forget this